### PR TITLE
add \r to content-update script, think it's needed for the release note tooling

### PR DESCRIPTION
**Pull request by jim-minter**
```release-note
NONE
```

### CODE REVIEW

File: hack/periodic/content-update.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ git reset HEAD^
 curl -u openshift-azure-robot:$GITHUB_TOKEN -H "Content-Type:application/json" -d @- -so /dev/null https://api.github.com/repos/openshift/openshift-azure/pulls <<'EOF'
 {
     "title": "Content update",
-    "body": "```release-notes\nNONE\n```",
+    "body": "```release-note\r\nNONE\r\n```",
```
Comment (asalkeld) [inline hack/periodic/content-update.sh]: did you mean to take the "s" off the release-notes?

**Comment by charlesakalugwu** [comment]
upgrade-v5.1 flake #1628
upgrade-v5.2 flake #1628
unit flake #1803

File: hack/periodic/content-update.sh (hunk 1 above)
Comment (charlesakalugwu) [reply hack/periodic/content-update.sh]: should be 
```suggestion
    "body": "```release-notes\r\nNONE\r\n```",
```

**Pull request merged**